app/src/main/java/com/framgia/soundclound/util/HomeBindingUtils.java

```diff
@@ -0,0 +1,78 @@
+package com.framgia.soundclound.util;
+
+import android.content.Context;
+import android.databinding.BindingAdapter;
+import android.support.constraint.ConstraintLayout;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.GridLayoutManager;
+import android.support.v7.widget.LinearLayoutManager;
```

Comment:
ERROR: Unused import: android.support.v7.widget.LinearLayoutManager. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports